src/common/hooks/use-network.ts

```diff
@@ -25,13 +43,33 @@ export const useNetwork = () => {
     [list, handleAddListItem, handleUpdateCurrentIndex]
   );
 
+  const handleRemoveNetwork = useCallback(
+    (item: { label: string; url: string }) => {
+      handleRemoveListItem(item);
+    },
+    [handleRemoveListItem]
+  );
+
+  const handleSetTestnet = useCallback(() => {
+    handleUpdateCurrentIndex(DEFAULT_TESTNET_INDEX);
+    router.reload();
+  }, []);
+
+  const handleSetMainnet = useCallback(() => {
+    handleUpdateCurrentIndex(DEFAULT_MAINNET_INDEX);
+    router.reload();
+  }, []);
+
   return {
     list,
     setList,
     index,
```

Comment:
`list` and `index` should definitely be exported as something more recognizable, like `networkList`, `currentNetworkIndex`, etc. I personally had to scroll up when reading a consumer of this variable to see what it was.